ServiceDocumentSamples/.prettierignore

```diff
@@ -7,4 +7,8 @@
 .sfdx
 .vscode
 
+# Ignore apex cls and apex trigger files:
+**/classes/FireCreateServiceDocumentInvocableAction.cls
```

Comment:
You added these to the ignore file, but then allowed Prettier to format them anyway. If Prettier formatting is working out for these files, I'd suggest to just allow them to be formatted, rather than adding entries to `.prettierignore`.

If you _are_ going to stick with the formatting exceptions, I'd propose you should do it for all of the files of these types rather than specific instances of them, e.g. `**/*.cls`. The decision to exempt files from formatting would typically be borne out of a desire not to do formatting for specific file types.

Reply:
prettier is working with these files, and I do have format them using prettier,
but, I have to install a version of 18 or higher version of java jdk.
So, I ignore them in case the customer have this dependency confilct.